### PR TITLE
stars created

### PR DESCRIPTION
**Pull request by lefteris-L**
[DEMO LINK](https://lefteris-l.github.io/layout_stars/)
[TEST REPORT LINK](https://lefteris-l.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,76 @@
...
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+    </div>
+
+    <div class="stars stars--1">
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
```
Comment (mgrinko) [inline src/index.html:26]: Don't add empty line if siblings are single line
Just is each sibling element has several lines

```suggestion
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
      
      <div class="stars stars--1">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```
Comment (mgrinko) [inline src/style.css:14]: Sizes are more important than images
So I'd put them first

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > .stars__star {
```
Comment (mgrinko) [inline src/style.css:22]: you could use :nth-child for all of then for standardization

**Review by mgrinko — APPROVED** [review]

File: src/style.css (hunk 3 above)
Comment (lefteris-L) [reply src/style.css:22]: Noted. I wasn't sure about the spaces. Thank you @mgrinko

**Pull request closed without merging**